<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>startandstop</class>
 <widget class="QWidget" name="startandstop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>181</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="Mypushbutton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Mypushbutton</class>
   <extends>QPushButton</extends>
   <header>my_pushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
